File: makediv3.py
# question => https://www.codechef.com/LP2TO301/problems/MAKEDIV3

t = int(input())
for i in range(t):
    n = int(input())
    for x in range(1,n+1):
        if(x == 1 or x == n):
            print('3',end="")
        else:
            print('0',end="")
    print("\n")

File: makediv3.cpp
// question => https://www.codechef.com/LP2TO301/problems/MAKEDIV3

#include <iostream>
using namespace std;

int main() {
	int t;
	cin>>t;
	while(t--){
	    int n;
	    cin>>n;
	    for(int i=1;i<=n;i++){
	        if(i==1 || i==n)
	            cout<<'3';
	        else cout<<'0';
	    }
	    cout<<endl;
	}
	return 0;
}
